fetch.md

```diff
@@ -0,0 +1,11 @@
+# Prerendering fetching modes
+
+Placeholder. TODO.
+
+Some text from Jeremy's doc:
+
+First, consider the __fetch__ of the resource. User agents would ideally prefetch the content in a way that does not identify the user. For example, the user agent could:
```

Comment:
I'm not sure about separating this so much from the discussion of what the consequences of this are. opt-in.md still discusses it a little, but in a way that might be too cursory to be useful to understand without clicking through.

Reply:
The idea is that this will eventually grow to a more detailed explanation of the changes and how they flow through the system. As such, these seem like the kernel of such a detailed description, and I moved them here.

My intention for opt-in.md is that you're supposed to arrive at that document taking for granted that an opt-in is needed, and if you want to find out why, then you'll have to click through to fetch.md or maybe README.md. (Probably some of https://github.com/WICG/portals#privacy-threat-model-and-restrictions needs to make its way here...)